### DISCUSSION TITLE
Allow application of shaders to the script editor.

### DISCUSSION THREAD

**Discussion by PabloDevGit** [godotengine/godot-proposals#6459] [Editor]
### Describe the project you are working on

It's not a project, it's just an idea.

### Describe the problem or limitation you are having in your project

I think many users would like to be able to give their code editor a retro look, or at least the code itself so that it looks like you're typing on a retro monitor.

Example, An example, I would like to give a CRT look to my code editor.
https://youtu.be/OvkRROqKK4k?t=1520

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Apply a shader to the script editor window, so that the text takes on different aspects through the shader.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

A shader applied to the code editor window.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No.

### Is there a reason why this should be core and not an add-on in the asset library?

Not sure.

**Comment by RedMser** [comment]
I think this could be done by adding a material overlay to the code editor node, but it would probably affect more than just the text (e.g. autocomplete?).

But probably possible with an add-on.

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.
